Sources/Testing/Traits/TimeLimitTrait.swift

```diff
@@ -15,8 +15,32 @@
 /// - ``Trait/timeLimit(_:)``
 @available(_clockAPI, *)
```

Comment:
Not for this PR, but we could remove this constraint now I think if we store a `TimeValue` instead of a `Swift.Duration`.